src/GameSlice.js

```diff
@@ -74,6 +71,14 @@ export const GameSlice = createSlice({
       state.currentSeconds += 1;
     },
     getDailyKryptoSuccess: (state, action) => {
+      if (state.kryptoId !== action.payload.id) {
```

Comment:
this checks if it is a new day with new krypto data, and will reset relevant data. Was not being reset before